### ISSUE TITLE
Serversecure doesn't load after a changelevel

### ISSUE THREAD

**Issue by uRandomAlex** [danielga/gmsv_serversecure#25]
The module works just fine when I start the server, but when I do a changelevel, I get this error:

```
[ERROR] unable to create detour for CNetChan::IsValidFileForTransfer
  1. unknown - [C]:-1
   2. require - [C]:-1
    3. unknown - lua/includes/modules/serversecure.lua:1
     4. unknown - [C]:-1
      5. pcall - [C]:-1
       6. unknown - lua/autorun/server/sv_serversecure.lua:32

Module not found!
```

My server runs on linux.

**Comment by uRandomAlex** [comment]
I also can't connect to the server after this error. It just shows "Connection failed after x retries".

**Comment by danielga** [comment]
Fixed on #26.
